src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```
@@ -1075,6 +1075,46 @@ properties:
             type: integer
             fits_keyword: DETYSIZ
             blend_table: True
+          multistripe_reads1:
+            title: Num. substripe rows read starting at SUBSTRT1
+            type: number
+            fits_keyword: MSTR_RD1
+            blend_table: True
+          multistripe_skips1:
+            title: Num. rows to skip after reading MSTR_RD1 rows
+            type: number
+            fits_keyword: MSTR_SK1
+            blend_table: True
+          multistripe_reads2:
+            title: Num. rows read in subsequent substripes
+            type: number
+            fits_keyword: MSTR_RD2
+            blend_table: True
+          multistripe_skips2:
+            title: Num. rows to skip between susequent substripes
+            type: number
+            fits_keyword: MSTR_SK2
+            blend_table: True
+          repeat_stripe:
+            title: Repeat reading MSTR_RD2 rows at first position
+            type: number
+            fits_keyword: MSTR_RPT
+            blend_table: True
+          interleave_reads1:
+            title: Interleave MSTR_RD1 rows between substripes
+            type: number
```

```suggestion
            type: integer
```
Same as repeat_stripe.